cl_billing_pricing.module

```diff
@@ -730,6 +802,12 @@ function cl_billing_pricing_check($action, $li_wrapper) {
 
 /**
  * Check a usage group action for a line item.
+ *
+ * @param $action array
```

Comment:
It might be useful to rename all of these variables to something like `$pricing_field_item` so that it is distinct language from the `action` sub-component of the field. I know this is my fault for not being clear in the original code.